Automattic/woocommerce-payments
# Update dev mode to Sandbox mode throughout the plugin

Fixes #7974

#### Changes proposed in this Pull Request





This PR updates all **front-facing** references in the plugin to "dev mode" to say "sandbox mode" which is aimed at making it more clear what the intention of this mode is, and to emphasise that live transactions can never occur in this mode. The full discussion is contained in the P2 thread and the latest designs: paJDYF-bGl-p2#comment-22402
 
I opted against updating code/comments in general because the concept of "dev mode" is quite firmly embedded into the plugin code and I think it's well understood by devs - the main purpose of this change is that the term sandbox mode is more easily understandable to front-end users of the plugin.





#### Testing instructions





* Onboard an account in sandbox mode (select this option when onboarding to be taken to the Stripe KYC flow).
* After onboarding, check each of the admin pages in WooPayments - anywhere where we'd previously have referred to dev mode should now instead reference sandbox mode.

TODO Post Merge

Docs will need to be updated prior to the next release. (Will chase this after PR is approved). https://woo.com/document/woopayments/testing-and-troubleshooting/dev-mode/

-------------------


**Post merge**
